recipes/libmagic/all/conanfile.py

```diff
@@ -0,0 +1,103 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.build import cross_building
+from conan.tools.env import VirtualBuildEnv, VirtualRunEnv
+from conan.tools.files import copy, get, rm, rmdir, rename
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.52.0"
+
+class LibmagicConan(ConanFile):
+    name = "libmagic"
+    description = "Magic number recognition library - detect files with data in particular fixed formats."
+    license = "DocumentRef-COPYING:LicenseRef-BSD-2-Clause-File"  # Modified BSD 2-Clause that states no restrictions on US export
+    homepage = "https://github.com/file/file"
```

Comment:
This should be the official home page.

```suggestion
    homepage = "https://www.darwinsys.com/file/"
```

Reply:
Thanks!